code/modules/surgery/organs/heart.dm

```diff
@@ -51,11 +51,11 @@
 
 /obj/item/organ/heart/on_life()
 	if(owner.client)
-	var/sound/slowbeat = sound('sound/health/slowbeat.ogg')
-	slowbeat.repeat = 1
+		var/sound/slowbeat = sound('sound/health/slowbeat.ogg')
+		slowbeat.repeat = 1
```

Comment:
This can be in 1 line var/sound/slowbeat = sound('sound/health/slowbeat.ogg', repeat = TRUE)